### ISSUE TITLE
Stop button not showing in Floorplan for Venetian Blinds EU

### ISSUE THREAD

**Issue by quack3d** [domoticz/domoticz#2323]
The Stop button on Venetian Blinds EU works fine from Switches tab but doesn't exist in the Floorplan.

**Comment by gizmocuz** [comment]
Unfortunately the initial creator of the floorplan did not support 3 images, only 2.
Therefor blinds can have at the moment no stop button
This has to be implemented

**Comment by dnpwwo** [comment]
I don't have any devices that have 3 buttons which made it hard to implement & test.  If someone posts what it should look like and can have a look at it.

**Comment by quack3d** [comment]
Here is the default theme: 
![image](https://user-images.githubusercontent.com/25592497/41292697-93e00f76-6e53-11e8-8612-e26b4a3bff96.png)

This is the one I use (Aurora): 
![image](https://user-images.githubusercontent.com/25592497/41292707-9d9189dc-6e53-11e8-9dd0-09f71e0d5fb1.png)

**Comment by gizmocuz** [comment]
Yes that does not help users using the default theme, not feature blinds that need to have a stop button.
This needs a general approach, not a hack in a theme. (also needed for mysensors)

**Comment by gryzli133** [comment]
Any progres with the stop button? What about venetian blinds with stop and percentage slider, would it be possible to implement?

**Comment by gizmocuz** [comment]
Should already been fixed over the years
